capstone_project/src/main/webapp/pages/mapsTemplate.html

```diff
@@ -20,6 +18,21 @@ <h1>Maps Feature</h1>
       <br>
       <div id="map"></div>
       <br>
+      <h3>Don't see your college?</h3>
+      </p>Submit your school's info below and it'll appear on the map!</p>
+      <form action="/school-data" method="POST">
+        <p>School Name:</p>
+        <textarea name= "name-input" required></textarea>
+        <br/><br/>
+        <p>School Latitude:</p>
+        <textarea name= "latitude-input" required></textarea>
+        <br/><br/>
+        <p>School Longitude:</p>
+        <textarea name= "longitude-input" required></textarea>
```

Comment:
For latitude and longtitude, since these are always going to be numbers, consider using the [input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number) tag and setting it to numbers only. PS: I don't think the max and min attributes in the example I gave are mandatory